alpine-rails-nginx/install-nginx.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
+
+NGINX_HEADERS_MORE_VERSION=0.261
+NGINX_VERSION=1.9.5
+
+function verified_curl {
+  url="$1"
+  file="$2"
+  sha256_hash="$3"
+  curl --silent --fail --location "$url" >| "$file" \
+    && echo "$sha256_hash  $file" | sha256sum -cs \
+    && tar xf "$file"
+}
+
+mkdir /usr/local/src
+cd /usr/local/src
+
+verified_curl \
+  "http://nginx.org/download/nginx-$NGINX_VERSION.tar.gz" \
+  "nginx-$NGINX_VERSION.tar.gz" \
+  '48e2787a6b245277e37cb7c5a31b1549a0bbacf288aa4731baacf9eaacdb481b'
+
+verified_curl \
+  "https://github.com/openresty/headers-more-nginx-module/archive/v$NGINX_HEADERS_MORE_VERSION.tar.gz" \
+  "headers-more-nginx-module-$NGINX_HEADERS_MORE_VERSION.tar.gz" \
+  '03d1f5fbecba8565f247d87a38f5e4b6440b0a56d752bdd2b29af2f1c4aea480'
+
+cd nginx-$NGINX_VERSION
+
+# Remove server name from error pages
+sed -i 's/\("<hr><center>nginx<\/center>" CRLF\)/\/* \1 *\//' -- src/http/ngx_http_special_response.c
```

Comment:
> Unfortunately there is no configuration option. This is hard-coded into the C file.

Thats stupid. But we have to accept it for now. I wounder if a patch, instead of an sed command might be more stable / deterministic?